ui/src/containers/organisations.list/billingaccounts/subscriptions/columns.tsx

```diff
@@ -54,14 +51,23 @@ export const getColumns: (
       meta: {
         headerFilter: false,
       },
-      cell: (info) =>
-        new Date(info.getValue() as Date).toLocaleString("en", {
-          month: "long",
-          day: "numeric",
-          year: "numeric",
-        }),
+      cell: (info) => getFormattedDateString(info.getValue()),
 
       footer: (props) => props.column.id,
     },
+    {
+      header: "Cancellation date",
+      accessorKey: "canceled_at",
+      meta: {
+        headerFilter: false,
+      },
+      cell: (info) => getFormattedDateString(info.getValue()),
+    },
+    {
+      header: "Status",
+      accessorKey: "state",
+      cell: (info) => capitalizeFirstLetter(info.getValue()),
+      filterVariant: "text",
```

Comment:
add a `select` filterVariant  instead of `text`

Reply:
Done